chapter-6.qmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "Data tidying"
+---
+
+## Notes
+
+Using prose, describe how the variables and observations are organised in each of the sample tables.
+
+
+##Solutions
+
+
+```{r}
+library(tidyverse)
+```
+
+## 6.2.1 Exercises
+
+1.Using prose, describe how the variables and observations are organised in each of the sample tables.
+
+2.  
```

Comment:
Some of these questions are not answered. Can you organize it so that each part is answered underneath its label?